src/routes/index.svelte

```diff
@@ -33,52 +34,58 @@
   const style = `vertical-align: text-top; margin-right: 5pt;`
 </script>
 
-<h1>
-  <img src="/name.svg" alt="Studenten bilden Schüler" width="1924px" height="163px" />
-</h1>
+{#if $microcopy?.country == 'de'}
+  <h1>
+    <img src="/name.svg" alt="Studenten bilden Schüler" width="1924px" height="163px" />
```

Comment:
maybe
```svelte
<h1>
{#if $microcopy?.indexImage}
<img><!-->...<!--></img>
{:else}
<!-->default<!-->
{/if}
</h1>

```

Reply:
I don't exactly know why the image name.svg is placed inside the git repo and not on contentful. When it would be on contentful it might be more elegant this way

Reply:
Yep, I also think so.

Reply:
Sounds good to me. In the next PR?